Cargo.toml

```
@@ -1,42 +1,48 @@
 [package]
 edition = "2021"
 name = "aruna_server"
-version = "1.0.0-rc.5"
+version = "1.1.0-rc.1"
```

```suggestion
version = "1.0.0"
```
we should start with 1.0.0.

---

Fixed in https://github.com/ArunaStorage/ArunaServer/pull/76/commits/bd3081fbebe63c9e408d19a46964197a4918b820